ci/qunit-selenium-runner.rb

```diff
@@ -1,22 +1,81 @@
 # frozen_string_literal: true
 
-require "qunit/selenium/test_runner"
```

Comment:
Is the name of this file still relevant given we're no longer using qunit-selenium after this? This can be done in a follow up though :)

Reply:
I think so. We are still running QUnit tests through Selenium.

Reply:
But didn't you remove `qunit-selenium` from the Gemfile?

Reply:
Yeah, that gem was just a wrapper that did results processing. The tests are still qunit tests run through a selenium webdriver, so I think this filename is good enough.